opencraft/settings.py

```diff
@@ -411,3 +415,7 @@
 
 # Limit the number of log entries fetched for each instance, for performance
 LOG_LIMIT = env.int('LOG_LIMIT', default=10000)
+
+# Subdomain blacklist #########################################################
+
+SUBDOMAIN_BLACKLIST = env.list('SUBDOMAIN_BLACKLIST', default=[])
```

Comment:
@smarnach That would work for me as a start - we can always add to it as we think of new ones.